gammapy/maps/axes.py

```diff
@@ -1153,6 +1188,7 @@ def from_table(cls, table, format="ogip", idx=0, column_prefix=""):
                 edges = edges_from_lo_hi(e_min, e_max)
                 axis = MapAxis.from_energy_edges(edges)
         elif format == "gadf-sed-norm":
+            # TODO: guess interp here
```

Comment:
There is no guess here, is not it?
The ToDo is maybe not pertinent anymore...

Reply:
I think the TODO is still valid: I noticed that the GADF sed format does not define the spacing / interp between the norm values. This is in general fine, however in some case this leads to the behaviour that the norm values are log spaced, but the interp keyword remains "lin". One way to work around this is to guess the interp type from the actual spacing between the node values. But in general this is a detail...